23-07-workshop/in-progress/query_region_plot_HR_plot_lc.ipynb

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
<p>Since we're not selecting a cluster, should we add a note that we've just arbitrarily selected some coordinates?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/23/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>Added "We're arbitrarily choosing a large patch on the sky to see the overall population of stars that we can find."</p>